src/microbe_stage/editor/PatchMapPopulationIndicator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Godot;
+
+/// <summary>
+///   Dots scattered around patch map nodes, indication population of player species
+/// </summary>
+public partial class PatchMapPopulationIndicator : Control
```

Comment:
There's going to be a ton of these, so I think that actually this shouldn't be a node type with a script attached. And instead just I'd inline this code to the patch map drawer class (though it should be cleanly put into a separate section as it is already a pretty long file). I think the performance impact will be much less when there's just simple graphics used for the indicators rather than nodes with scripts attached.